### PR TITLE
Expand FAQ info about port usage

### PR DESCRIPTION
**Pull request by bboreham**

### CODE REVIEW

**Comment by awh** [comment]
#### Documentation changes:
modified [faq](https://dev-weavewww.pantheonsite.io/docs/net/ports-faq/faq/)

File: site/faq.md [hunk 1]
```diff
@@ -90,6 +90,13 @@ Yes, of course!  Weave allows you to run isolated networks and still allow open
 You must permit traffic to flow through TCP 6783 and UDP 6783/6784,
 which are Weave’s control and data ports.
 
+The daemon also uses port 6782 for [metrics](/site/metrics.md), but
```
Comment (brb) [inline site/faq.md]: "port 6782" -> "TCP port 6782"

File: site/faq.md [hunk 2]
```diff
@@ -90,6 +90,13 @@ Yes, of course!  Weave allows you to run isolated networks and still allow open
...
+you would only need to open up this port if you wish to collect metrics
+from another host.
+
+The Weave Net daemon listens on localhost (127.0.0.1) tcp port 6784
```
Comment (brb) [inline site/faq.md]: Nitpick: s/tcp/TCP

**Comment by brb** [comment]
Could you change the base branch to `1.9`?

**Comment by awh** [comment]
#### Documentation changes:
modified [faq](https://dev-weavewww.pantheonsite.io/docs/net/ports-faq/faq/)

**Comment by bboreham** [comment]
Updated, rebased and changed branch.

**Pull request merged**